src/dfx/src/lib/project/import.rs

```diff
@@ -239,16 +239,10 @@ impl Loader {
 
     fn client(&mut self) -> Result<&Client, ProjectError> {
         if self.client.is_none() {
-            let tls_config = rustls::ClientConfig::builder()
-                .with_safe_defaults()
-                .with_webpki_roots()
-                .with_no_client_auth();
-
             // Advertise support for HTTP/2
             //tls_config.alpn_protocols = vec![b"h2".to_vec(), b"http/1.1".to_vec()];
```

Comment:
These comments should probably be deleted as they are clutter now.

Reply:
thanks, wasn't sure what they were adhering to